core/src/main/java/lucee/runtime/db/HSQLDBHandler.java

```diff
@@ -260,8 +263,19 @@ public QueryImpl execute(PageContext pc, final SQL sql, int maxrows, int fetchsi
 			catch (PageException ex) {}
 
 		}
-		catch (PageException e) {}
-		// if(true) throw new RuntimeException();
+		catch (PageException e) {
+			qoqException = e;
+		}
+
+		// Debugging option to completely disable HyperSQL for testing
+		if (qoqException != null && Caster.toBooleanValue(SystemUtil.getSystemPropOrEnvVar("lucee.qoq.hsqldb.disable", "false"))) {
```

Comment:
@michaeloffner The reason I was checking it every time was because during my tests I would toggle the setting back and forth between tests without restarting the server.  I can make this a static variable though if we think users will not do that.

Reply:
make sense and i agree, in other places env var are only read once, so a restart always is necessary.